app/models/infra_conversion_job.rb

```diff
@@ -104,6 +104,11 @@ def poll_conversion
 
     case v2v_status
     when 'active'
+      begin
+        migration_task.apply_virtv2v_limits if migration_task.options.fetch_path(:virtv2v_wrapper, 'throttling_file')
```

Comment:
Where is `throttling_file` coming from? Is that a configurable thing? Or is hard coding it ok?

Reply:
It comes from the output of virt-v2v-wrapper, like `state_file`. So, pretty much hard coded.

Reply:
@fdupont-redhat, 2 question
* are we doing this `apply_virtv2v_limits` in every polling, like every 15s, regardless limits changed or not?
* so this control flow is `infra_conversion_job -> miq_request_task.apply_virtv2v_limits -> conversion_host.apply_virtv2v_limits`? seems convoluted. This is the reason I asked to re-consider where we want to keep the throttling logic.

Reply:
Agree with @jameswnl Seems like too many hoops.